### PR TITLE
vote-styling

### LINKED ISSUE #69: Poll-vote page doesn't show the title of the poll
Maybe an error that slipped in in a recent PR that had some old content. Should've been caught in a test 😕

### PR DESCRIPTION
**Pull request by thysruben**
fixes #69

### CODE REVIEW

File: src/main/resources/templates/polls/detail.twig [hunk 1]
```diff
@@ -4,28 +4,44 @@
...
+				<input type="text" name="option1" id="option1" required/>
+				{% if errors.getFieldError("options") %}
+					<p class="error">{{ errors.getFieldError("options").getDefaultMessage() }}</p>
+				{% endif %}
+			</div>
+		</div>
+		<div class="poll--option">
+			<input type="submit" value="create" />
+		</div>
+		<details class="poll--advanced" {% if errors.getFieldError("slug") %}open{% endif %}>
```
Comment (Haroenv) [inline src/main/resources/templates/polls/detail.twig]: Why does the vote page have the advanced options?

File: src/main/resources/templates/polls/detail.twig [hunk 2]
```diff
@@ -4,28 +4,44 @@
...
 {% block content %}
-	<h1>{{ poll.name }}</h1>
-
-	<form method="post" action="/polls/vote/{% if poll.slug is empty %}{{ poll.id }}{% else %}{{ poll.slug }}{% endif %}" class="poll">
-		{%- for option, nVotes in poll.options %}
-		<div class="poll--vote">
-			<label for="{{ option }}">{{ option }}: {{ nVotes }}</label>
-			<input type="radio" name="option" id="{{ option }}" value="{{ option }}">
+	<form method="post" action="/polls/create" class="poll" id="createPollForm">
+		<div class="poll--option poll--title">
```
Comment (Haroenv) [inline src/main/resources/templates/polls/detail.twig]: Tabs?

File: src/main/resources/templates/polls/detail.twig [hunk 3]
```diff
@@ -4,28 +4,32 @@
 {% block description %}{{ poll.title }} {{ poll.options }}{% endblock %}
 
 {% block content %}
-	<h1>{{ poll.name }}</h1>
+	<h1>{{ poll.title }}</h1>
 
-	<form method="post" action="/polls/vote/{% if poll.slug is empty %}{{ poll.id }}{% else %}{{ poll.slug }}{% endif %}" class="poll">
-		{%- for option, nVotes in poll.options %}
+	<div class="poll">
```
Comment (Haroenv) [inline src/main/resources/templates/polls/detail.twig]: I'd use another class for this. The poll class will get polluted. .vote might be a better choice

File: src/main/resources/templates/polls/detail.twig [hunk 4]
```diff
@@ -4,28 +4,32 @@
...
-		{%- endfor %}
-		<button type="submit">Vote</button>
 	</form>
 	<form method="post" action="/polls/delete/{{ poll.id }}">
-		<input type="submit" value="delete">
-		<input type="hidden" name="{{ _csrf.parameterName }}" value="{{ _csrf.token }}"/>
+		<div class="poll--delete">
+			<input type="submit" value="delete">
+			<input type="hidden" name="{{ _csrf.parameterName }}" value="{{ _csrf.token }}"/>
+		</div>
```
Comment (Haroenv) [inline src/main/resources/templates/polls/detail.twig]: Can be ignored here, will be moved

File: src/main/resources/templates/polls/detail.twig [hunk 5]
```diff
@@ -4,28 +4,32 @@
...
 	</form>
-	<div class="social" id="social">
+	<div class="poll--social">
```
Comment (Haroenv) [inline src/main/resources/templates/polls/detail.twig]: Social isn't part of .poll, since it's also used in results for example.

**Comment by thysruben** [comment]
screenshot 2016-12-11 21 50 39

File: src/main/resources/templates/polls/detail.twig [hunk 6]
```diff
@@ -4,28 +4,32 @@
...
 		<a href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fpollistics.com%2F{% if poll.slug is empty %}{{ poll.id }}{% else %}{{ poll.slug }}{% endif %}&quote={{poll.title|url_encode}}" target="_blank" rel="noopener" class="cta">share on facebook</a>
 		<a href="https://twitter.com/intent/tweet?text={{ poll.title ~ "\n\n" ~ "via Pollistics — https://pollistics.com/" | url_encode }}{% if poll.slug is empty %}{{ poll.id }}{% else %}{{ poll.slug }}{% endif %}" target="_blank" rel="noopener" class="cta">share on twitter</a>
 	</div>
+	<div class="poll--link" id="social"></div>
```
Comment (Haroenv) [inline src/main/resources/templates/polls/detail.twig]: I don't get why you broke it in two

**Review by Haroenv — COMMENTED** [review]
Not so sure

**Comment by Haroenv** [comment]
You shouldn't see the number of votes when voting though. That'll pollute your thinking

File: src/main/resources/templates/polls/detail.twig (hunk 6 above)
Comment (hannnegoossens) [reply src/main/resources/templates/polls/detail.twig]: we did this because we wanted the 2 social buttons flexed, was easier like this

Comment (Haroenv) [reply src/main/resources/templates/polls/detail.twig]: Hm. Should be different IMO

Comment (hannnegoossens) [reply src/main/resources/templates/polls/detail.twig]: We think so too but do you know a way to change this without losing the styling of the facebook and twitter buttons?

Comment (Haroenv) [reply src/main/resources/templates/polls/detail.twig]: should be solved as a separate issue in gin-fizz/styleguide#11

File: src/main/resources/templates/polls/detail.twig [hunk 7]
```diff
@@ -6,18 +6,20 @@
...
-		</div>
-		<div>
-			<input type="hidden" name="{{ _csrf.parameterName }}" value="{{ _csrf.token }}"/>
-		</div>
-		{%- endfor %}
-		<button type="submit">Vote</button>
-	</form>
+	<div class="vote">
+		<form method="post" action="/polls/vote/{% if poll.slug is empty %}{{ poll.id }}{% else %}{{ poll.slug }}{% endif %}">
+			<div class="vote--options">
```
Comment (Haroenv) [inline src/main/resources/templates/polls/detail.twig]: reuse a syntax like with polls here.

**Review by Haroenv — COMMENTED** [review]
Goed werk brave man

**Review by Haroenv — APPROVED** [review]
Guud man

**Pull request merged**